modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java

```diff
@@ -256,21 +255,22 @@ public String[] getFileVersions(
 			_blobContainerClient.listBlobsByHierarchy(
 				StringPool.SLASH, listBlobsOptions, null);
 
-		Stream<BlobItem> stream = pagedIterable.stream();
+		List<String> fileVersions = new ArrayList<>();
 
-		return stream.filter(
-			blobItem -> !GetterUtil.getBoolean(blobItem.isPrefix())
-		).map(
+		pagedIterable.forEach(
 			blobItem -> {
+				if (!GetterUtil.getBoolean(blobItem.isPrefix())) {
+					return;
+				}
+
 				String blobItemName = blobItem.getName();
 
-				return blobItemName.substring(prefix.length());
-			}
-		).sorted(
-			DLUtil::compareVersions
-		).toArray(
-			String[]::new
-		);
+				fileVersions.add(blobItemName.substring(prefix.length()));
+			});
+
+		Collections.sort(fileVersions, DLUtil::compareVersions);
```

Comment:
@jeffwu0724 Please directly use `List.sort` as IntelliJ IDEA recommends. You can search for the difference between these two.